### PR TITLE
Blog Post: Eighth Wave of Nostr Grants

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build Preview:**

- [/blog/8th-wave-nostr-grants](https://os-website-git-8th-wave-nostr-open-sats.vercel.app/blog/8th-wave-nostr-grants)

**Social Preview:**

- 
image

### CODE REVIEW

**Comment by Arvin21M** [comment]
@OpenSats/volunteers team, would some one mind giving it one more quick read as a sanity check for any last minute adjustments? 🙏

**Comment by ecurrencyhodler** [comment]
It says 4 projects but 5 are listed. Is that meant to be?

**Comment by Arvin21M** [comment]
> It says 4 projects but 5 are listed. Is that meant to be?

Fixed 🙏

**Comment by Arvin21M** [comment]
@dergigi ready for a final review, and then merge. 🫡

**Comment by ecurrencyhodler** [comment]
@Arvin21M can you do a search for "four" and replace it with "five"? You use it a couple of other times in the blog!

**Comment by Arvin21M** [comment]
> @Arvin21M can you do a search for "four" and replace it with "five"? You use it a couple of other times in the blog!

Fixed... again 😆 

Feel free to make suggested edits directly into the code file. You should be able to from your end next time, and I will just press a button to accept the change on my side. Here is [a helpful how-to](https://haacked.com/archive/2019/06/03/suggested-changes/) on this.

🙏

File: data/blog/8th-wave-nostr-grants.mdx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+Repository: [max21dev/groups](https://github.com/max21dev/groups)  
+License: MIT
+
+### Coop
+
+Coop is a messaging client built on nostr, designed for personal and group
+communication with a focus on gamers. It offers a Discord-like experience, while
+also enabling secure and simple communication through features such as group
+chats and, threaded discussions with plans for voice and video calls. It
```
Comment (julian2000-code) [inline data/blog/8th-wave-nostr-grants.mdx:86]: Delete comma after 'and'

**Review by julian2000-code — APPROVED** [review]

**Comment by btcschellingpt** [comment]
Looks great .. let that puppy bark out load!

**Comment by dergigi** [comment]
Thanks everyone for writing/editing/reviewing this one! Appreciate it 🙏🧡

Live now:
- https://opensats.org/blog/8th-wave-nostr-grants
- [nostr post](https://njump.me/nevent1qqstnfm6yr2lw6erkjde9kwdhf8yq64hxegqm60hvfz0eap3853c0qspp4mhxue69uhkummn9ekx7mqzypu8xwr40lp96ewdj2fef408wy70gd3carf9n6xu7hrnhq6whpglyssxwrg)

**Pull request merged**